docs/src/pages/demos/tables/CustomizedTable.js

```diff
@@ -16,7 +16,7 @@ const CustomTableCell = withStyles(theme => ({
   body: {
     fontSize: 14,
   },
-}))(TableCell);
+}))(props => <TableCell {...props} />);
```

Comment:
Related to https://github.com/mui-org/material-ui/issues/13776

Reply:
👍

Reply:
But TableCell is no forwardRef component. Why would this be an issue?

Reply:
@eps1lon withStyles uses it.